packages/plugin-webpack/plugin.js

```diff
@@ -1,5 +1,7 @@
 const fs = require("fs");
+const glob = require("glob");
 const path = require("path");
+const util = require("util");
```

Comment:
I think both of these have to be added as dependencies to `package.json`?

Reply:
I believe just `glob`. `util` should be a native lib

Reply:
🤦‍♂️ I feel dumb! I don’t think I’ve used that built-in before.

Reply:
That's alright! We all have those 🤦 moments! 😄

Reply:
oops! I've added this to package.json